.github/workflows/test.yml

```diff
@@ -349,10 +349,10 @@ jobs:
     needs: setup
     env:
       # Various feature flags
-      FF_TFM_PAYMENT_RECONCILIATION_ENABLED: 'true'
-      FF_TFM_FACILITY_END_DATE_ENABLED: 'true'
-      FF_TFM_DEAL_CANCELLATION_ENABLED: 'true'
-      FF_TFM_SSO_ENABLED: 'true'
+      FF_TFM_PAYMENT_RECONCILIATION_ENABLED: true
```

Comment:
Env vars are parsed as strings, so we should keep these as strings to maintain reliability in our tests (I'm not sure if github will pass these as strings or not magically, but we can be explicit to avoid having to research this please)

Reply:
@abhi-markan Do you mind if we remove this commit from this hotfix please as I feel it needs a bit more thought

Reply:
As discussed I will raise a separate PR. Thanks